naming/src/main/java/com/alibaba/nacos/naming/healthcheck/HealthCheckType.java

```diff
@@ -15,24 +15,49 @@
  */
 package com.alibaba.nacos.naming.healthcheck;
 
+import com.alibaba.nacos.api.naming.pojo.AbstractHealthChecker;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
 /**
  * @author nkorange
  */
 public enum HealthCheckType {
     /**
      * TCP type
      */
-    TCP,
+    TCP("tcp", AbstractHealthChecker.Tcp.class),
     /**
      * HTTP type
      */
-    HTTP,
+    HTTP("http", AbstractHealthChecker.Http.class),
     /**
      * MySQL type
      */
-    MYSQL,
+    MYSQL("mysql", AbstractHealthChecker.Mysql.class),
     /**
      * No check
      */
-    NONE
+    NONE("none", AbstractHealthChecker.None.class);
+
+    private String name;
```

Comment:
The name here is useless?

Reply:
Not currently, but I think it may be used in the future.